src/eckit/mpi/Comm.cc

```diff
@@ -95,10 +95,10 @@ class Environment {
         for (itr = communicators.begin(); itr != communicators.end(); ++itr) {
             eckit::Log::error() << "   " << (*itr).first << std::endl;
         }
-        throw eckit::SeriousBug(std::string("No communicator called ") + name, Here());
+        throw eckit::SeriousBug(std::string("No communicator called ") + std::string(name), Here());
```

Comment:
Being pedantic here, for even more terseness one can remove the first std::string construction (in more than one place I see)

Reply:
Yeah, that would still work. My nitpick comment is that I prefer braces '{}' for calling the constructor to make it immediately obvious that we are calling the constructor and not a free function. This used to be very helpful for not getting caught out by 'the most vexing parse' behaviour, but these days compilers would warn you about that anyway.

Otherwise, it looks good.

Reply:
Thanks! We aim for perfection :) I have implemented all suggestions.